src/sections/web3-connect/accounts/Web3ConnectAccountSelect.tsx

```diff
@@ -38,7 +37,7 @@ export const Web3ConnectAccountSelect = ({
   const isDesktop = useMedia(themeParams.viewport.gte.sm)
 
   return (
-    <ButtonTransparent onClick={onClick} sx={{ gap: 12 }}>
+    <div onClick={onClick} sx={{ flex: "row", align: "center", gap: 12 }}>
```

Comment:
Why change from button to div? 

onClick handlers are generally not good practice on divs

Reply:
There is a warning in console, that button cannot be inside another button element